can-observation.js

```diff
@@ -101,6 +101,11 @@ function Observation(func, context, compute){
 	this.childDepths = {};
 	this.ignore = 0;
 	this.needsUpdate= false;
+	if (!func) {
+		this.start = function(){
```

Comment:
An alternative to inserting the start method there, could be modifying `update()` to be:
```
	update: function(batchNum){
		if(this.needsUpdate) {
			remaining.updates--;
		}
		this.needsUpdate = false;
		if(this.bound) {
			// Keep the old value.
			var oldValue = this.value;
			this.oldValue = null;
			// Get the new value and register this event handler to any new observables.
			if (this.func) {
				this.start();
			}
			if(oldValue !== this.value || !this.func) {
				this.compute.updater(this.value, oldValue, batchNum);
				return true;
			}
		}
	},
```

Would that be a more preferable approach?